packages/api/src/command/medical/admin/get-hie-enabled-feature-flags-status.ts

```diff
@@ -0,0 +1,14 @@
+import { getFeatureFlags, CxFeatureFlagStatus } from "@metriport/core/external/aws/app-config";
+import { Config } from "../../../shared/config";
+
+export async function getHieEnabledFFStatus(cxId: string): Promise<CxFeatureFlagStatus> {
+  const region = Config.getAWSRegion();
+  const appId = Config.getAppConfigAppId();
+  const configId = Config.getAppConfigConfigId();
+  const envName = Config.getEnvType();
+  const featureFlags = await getFeatureFlags(region, appId, configId, envName);
```

Comment:
Could we be even more specific and just use this https://github.dev/metriport/metriport/blob/bddfef6535fb4b66b7a8c83c32e1b20d5d53a5a6/packages/api/src/external/aws/app-config.ts#L98-L101 ditto with cq one

Reply:
We need the full thing for updating as we have to over-write the whole thing.